modules/react-native-status/android/src/main/java/im/status/ethereum/pushnotifications/PushNotificationHelper.java

```diff
@@ -701,37 +695,31 @@ private Person getPerson(Bundle bundle) {
 
     private StatusMessage createMessage(Bundle data) {
         Person author = getPerson(data.getBundle("notificationAuthor"));
-        return new StatusMessage(data.getString("id"), author,  data.getLong("timestamp"), data.getString("message"));
+        long timeStampLongValue = (long) data.getDouble("timestamp");
```

Comment:
The following error is discovered using `adb logcat`

```java
Bundle  : Key timestamp expected Long but value was a java.lang.Double.  The default value 0 was returned.
Bundle  : Attempt to cast generated internal exception:
Bundle  : java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Long
                  at android.os.BaseBundle.getLong(BaseBundle.java:1289)
                  at android.os.BaseBundle.getLong(BaseBundle.java:1271)
                  at im.status.ethereum.pushnotifications.PushNotificationHelper.createMessage(PushNotificationHelper.java:704)
                  at im.status.ethereum.pushnotifications.PushNotificationHelper.addStatusMessage(PushNotificationHelper.java:765)
                  at im.status.ethereum.pushnotifications.PushNotificationHelper.handleConversation(PushNotificationHelper.java:193)
```
Added appropriate fix by type casting.